manifests/init.pp

```diff
@@ -15,6 +15,8 @@
 #
 # $proxy_password::     Proxy password for authentication
 #
+# $pulp_max_speed::     Maximum bandwidth used per download in bytes/sec from CDN
```

Comment:
Similar to my comment on the Pulp PR, if this only applies to yum I think it should indicate that, e.g. pulp_yum_max_speed